readthedocs/source/notation_and_other_cosmological_conventions.rst

```diff
@@ -249,7 +249,7 @@ basic physical constants
   - CLIGHT: speed of light in m/s
   - EV_IN_J: conversion factor between electron volts and Joules
   - T_CMB: temperature of the CMB in K
-  - TNCDM: temperature of the cosmological neutrino background in K
+  - T_ncdm: temperature of the cosmological neutrino background in K
```

Comment:
remove this and TCMB

Reply:
as above

Reply:
yes, so we can remove them from here so that people won't find them in the docs any more, and keep them for internal use maybe? what do you think?

Reply:
Unresolved this. So we agreed to remove them from the dics, right?

Reply:
yes, I removed them